lib/swagger/swagger.yaml

```diff
@@ -130,7 +130,62 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/GeneralError'      
+                $ref: '#/components/schemas/GeneralError'
```

Comment:
hmm, don't we give the ability to retrieve a wave file as well?

Reply:
You're right. I tried the /wav and it wasn't working and realised now that only if we changed the file format to "wav" in settings would the wav version of the recording be uploaded to s3. I will change this now.